code/_onclick/hud/screen_objects.dm

```diff
@@ -388,9 +389,13 @@
 
 	if(hud && hud.mymob)
 		if(iscarbon(hud.mymob))
-			var/mob/living/carbon/C = hud.mymob
+			var/mob/living/carbon/human/C = hud.mymob
```

Comment:
casting can go wrong. Due to you checking if it's carbon. Then casting it to human

Reply:
Good catch, not sure how I missed this!